### ISSUE TITLE
[Feature Request] Add an option to choose a custom Xenia executable for specific games via right-click or similar method

### ISSUE THREAD

**Issue by secluls** [xenia-manager/xenia-manager#70]
**Is your feature request related to a problem? Please describe.**

This would be a useful quality of life feature, as some games perform better with a custom Xenia version or by using a fork for a particular game, such as the Mousehook Xenia fork.

**Describe the solution you'd like**

By right-clicking on an added game, a context menu should appear, allowing you to choose which version of Xenia you want to use by selecting a Xenia executable from a menu.

**Additional context**

It would be cool if a star icon appeared in the top right corner of the game box to indicate that it's using a custom Xenia installation.

**Comment by shazzaam7** [comment]
I'll have to think about this, but if I were to do it, I would probably disable the option to Install things like patches and content for that game if it's using custom Xenia version.

Currently I'm focused on adapting the app UI to work in Fullscreen mode and this has already been requested by another user so I'll have to take a closer look.

Thanks for doing a proper "Feature Request".

**Comment by ArtemGT69** [comment]
> **Is your feature request related to a problem? Please describe.**
> 
> This would be a useful quality of life feature, as some games perform better with a custom Xenia version or by using a fork for a particular game, such as the Mousehook Xenia fork.
> 
> **Describe the solution you'd like**
> 
> By right-clicking on an added game, a context menu should appear, allowing you to choose which version of Xenia you want to use by selecting a Xenia executable from a menu.
> 
> **Additional context**
> 
> It would be cool if a star icon appeared in the top right corner of the game box to indicate that it's using a custom Xenia installation.

I agree with this because Forza Motorsport 2 or any forza game perform better in older builds than newer

**Comment by shazzaam7** [comment]
I have added an option to Switch to a Custom Version of Xenia in #84 and merged it into the dev branch.
You can test this in the new [experimental build](https://github.com/xenia-manager/xenia-manager/releases/tag/experimental).

**Comment by ArtemGT69** [comment]
> I have added an option to Switch to a Custom Version of Xenia in #84 and merged it into the dev branch. You can test this in the new [experimental build](https://github.com/xenia-manager/xenia-manager/releases/tag/experimental).

Okay

**Comment by ArtemGT69** [comment]
> I have added an option to Switch to a Custom Version of Xenia in #84 and merged it into the dev branch. You can test this in the new [experimental build](https://github.com/xenia-manager/xenia-manager/releases/tag/experimental).

its working fine
can i post video to show it?

**Comment by shazzaam7** [comment]
> its working fine can i post video to show it?
Of course, it is public after all.

**Comment by ArtemGT69** [comment]
> > its working fine can i post video to show it?
> 
> Of course, it is public after all.

this feature is useful i'm pretty sure
![firefox_0WYHHI1MN3](https://github.com/user-attachments/assets/3b90e089-0490-4dce-bef2-beb0da0c954f)
